cli/how-to-deploy-triton-ensemble-managed-online-endpoint.sh

```diff
@@ -0,0 +1,49 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+set -e
+
+# <set_endpoint_name>
+export ENDPOINT_NAME=triton-ensemble-mir-endpt-`echo $RANDOM`
+# </set_endpoint_name>
```

Comment:
You should download the model from some publicly accessible blob store instead of checking it into the repo